erpnext/regional/united_arab_emirates/utils.py

```diff
@@ -30,10 +53,16 @@ def update_itemised_tax_data(doc):
 				tax_amount += flt((row.net_amount * _tax_rate) / 100, row.precision("tax_amount"))
 				tax_rate += _tax_rate
 
+		if not tax_rate and has_zero_rated_field and not row.is_zero_rated:
```

Comment:
`not row.is_zero_rated` check could give unexpected results. eg: originally export, then customer or customer address changed.